src/middlewared/middlewared/api/v25_04_0/system_security.py

```diff
@@ -13,12 +18,52 @@
 ]
 
 
+PASSWORD_COMPLEXITY_CHOICES = Literal[
+    PasswordComplexity.UPPER,
+    PasswordComplexity.LOWER,
+    PasswordComplexity.NUMBER,
+    PasswordComplexity.SPECIAL,
+]
+
+
 class SystemSecurityEntry(BaseModel):
     id: int
     enable_fips: bool
     """ When set, enables FIPS mode. """
     enable_gpos_stig: bool
     """ When set, enables compatibility with the General Purpose Operating System STIG. """
+    min_password_age: Annotated[int, Ge(1)] | None = None
```

Comment:
You can just do
```
from pydantic import Field

min_password_age: int | None = Field(ge=1, default=None)

Reply:
I see that you're simply using the same paradigm that already exists in this file so I'll mark this as resolved. I've opened another ticket to be done later for removing the `annotated_types` module since it's not needed.